src/pyenphase/auth.py

```diff
@@ -69,18 +70,26 @@ async def setup(self, client: httpx.AsyncClient) -> None:
                 "Unable to obtain token for Envoy authentication."
             )
 
-        # Verify the token and obtain cookie with session ID necessary for some API calls
+        await self._check_jwt(client)
+
+    @retry(
+        retry=retry_if_exception_type(httpx.HTTPError),
+        wait=wait_random_exponential(multiplier=2, max=3),
```

Comment:
max=10 like the others ?

Reply:
I was a bit worried about blocking HA startup. The goal was to get it to fall into retry and eventually get going